cocos2d/Support/CCFile_Private.h

```diff
@@ -55,6 +55,13 @@ typedef NSInputStream *(^CCStreamedImageSourceStreamBlock)();
 @end
 
 
+@interface CCFile()
+
+@property (nonatomic, assign, readwrite) BOOL useUIScale;
+
+@end
+
+
 @interface CCFile(Private)
```

Comment:
Maybe this named category should have the name removed, and useUIScale could just go below?